src/libs/dutil/thmutil.cpp

```diff
@@ -1979,6 +1977,74 @@ static HRESULT ParseFonts(
 }
 
 
+static HRESULT GetFontColor(
+    __in IXMLDOMNode* pixn,
+    __in_z LPCWSTR wzAttributeName,
+    __out COLORREF* pColorRef,
+    __out_opt DWORD* pdwSystemColor
+    )
+{
+    HRESULT hr = S_OK;
+    BSTR bstr = NULL;
+
+    *pdwSystemColor = 0;
+
+    hr = XmlGetAttribute(pixn, wzAttributeName, &bstr);
+    if (S_FALSE == hr)
+    {
+        *pColorRef = THEME_INVISIBLE_COLORREF;
+        ExitFunction1(hr = S_OK);
+    }
+    ExitOnFailure(hr, "Failed to find font %ls color.", wzAttributeName);
+
+    if (pdwSystemColor && '$' == bstr[0])
```

Comment:
Let's ditch the special leading character, there are too many special characters in WiX already.